### ISSUE TITLE
"Can't sign: User interface error" SMIME signing

### ISSUE THREAD

**Issue by nscblauensteiner** [znuny/Znuny#61]
### Environment

- OS:           Ubuntu 18.04.5 LTS
- Browser:      Google Chrome 90.0.4430.85
- Znuny version: 6.0.33

### Expected behaviour

Notification mails are going to be SMIME signed.

### Actual behaviour

Notification are not going to be SMIME signed, with the following error in the main log (debug level activated):

OTRS-otrs.Console.pl-Maint::PostMaster::MailAccountFetch-10 Can't sign: User interface error

### How to reproduce

If any OTRS function is going to be triggered (customer change for a ticket, ticket close, ...) outgoing
mails are not going to be signed.
New tickets or an ticket answer i.e. are going to be signed correctly.

### Additional information

Main log:

```
[Wed Apr 21 09:45:38 2021][Error][Kernel::System::Crypt::SMIME::Sign][457] Can't sign: User interface error
! (Command: smime -sign -in /opt/otrs/var/tmp/v_hs5zlrJ3.tmp -out /opt/otrs/var/tmp/E4zDHbTjdD.tmp -signer /opt/otrs/var/tmp/P67lMzFfkj.tmp -inkey /opt/otrs/var/tmp/dBng_LD3ka.tmp -text -binary -passin file:/opt/otrs/var/tmp/y4jmo5y43K.tmp)
[Wed Apr 21 09:45:38 2021][Info][Kernel::System::Ticket::Event::NotificationEvent::Transport::Email::SendNotification] Sent agent 'Ticket create notification' notification to '<USERNAME>@<MAIL_DOMAIN>'.

---

var/log/Daemon/SchedulerTaskWorkerERR-1618923152.log:


ERROR: OTRS-otrs.Console.pl-Maint::PostMaster::MailAccountFetch-10 Perl: 5.26.1 OS: linux Time: Tue Apr 20 14:50:24 2021

Message: Can't sign: User interface error
! (Command: smime -sign -in /opt/otrs/var/tmp/p7oyoVx617.tmp -out /opt/otrs/var/tmp/PW2XBj7wyb.tmp -signer /opt/otrs/var/tmp/kqoROBEXea.tmp -inkey /opt/otrs/var/tmp/M0evHvUfKS.tmp -text -binary -passin file:/opt/otrs/var/tmp/QSf14_B8Zw.tmp)

Traceback (31489):
Module: Kernel::System::Crypt::SMIME::Sign Line: 457
Module: Kernel::System::Email::Send Line: 455
Module: Kernel::System::Ticket::Event::NotificationEvent::Transport::Email::SendNotification Line: 189
Module: Kernel::System::Ticket::Event::NotificationEvent::_SendRecipientNotification Line: 1220
Module: Kernel::System::Ticket::Event::NotificationEvent::Run Line: 317
Module: Kernel::System::EventHandler::EventHandler Line: 223
Module: Kernel::System::EventHandler::EventHandlerTransaction Line: 263
Module: Kernel::System::ObjectManager::ObjectEventsHandle Line: 425
Module: Kernel::System::ObjectManager::ObjectsDiscard Line: 473
Module: Kernel::System::ObjectManager::DESTROY Line: 642
Module: (eval) Line: 209
Module: main::Start Line: 209
Module: /opt/otrs/bin/otrs.Daemon.pl Line: 152
```

---

### Screenshots

![image](https://user-images.githubusercontent.com/54094754/115701152-c5c65300-a367-11eb-9200-6196bda7802b.png)

**Comment by hanneshal** [comment]
@nscblauensteiner just a what is the value in the Sysconfig for this key: SMIME::Bin

![image](https://user-images.githubusercontent.com/804939/115707344-f362ca80-a36e-11eb-8d19-83b261616104.png)

Can you try to manually sign using the command from the log? What is the output?

Regards
Johannes

**Comment by rkaldung** [comment]
@nscblauensteiner Does the sender address when you reply to a ticket is the same as used for the other e-mails?

**Comment by nscblauensteiner** [comment]
> @nscblauensteiner just a what is the value in the Sysconfig for this key: SMIME::Bin
> 
> ![image](https://user-images.githubusercontent.com/804939/115707344-f362ca80-a36e-11eb-8d19-83b261616104.png)
> 
> Can you try to manually sign using the command from the log? What is the output?
> 
> Regards
> Johannes

Value:

![image](https://user-images.githubusercontent.com/54094754/115710808-0d9ea780-a373-11eb-8228-173bcf79399d.png)

**Comment by nscblauensteiner** [comment]
> Does the sender address when you reply to a ticket is the same as used for the other e-mails?

Yes

**Comment by hanneshal** [comment]
Additional Note: Error first noted in 6.0.30
Remote Call scheduled

**Comment by hanneshal** [comment]
fixed after update to 6.0.34. 
We did not find the cause, but it is working fine now.
